pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -135,6 +135,23 @@ type PrefilterPlugin interface {
 	Prefilter(pc *PluginContext, p *v1.Pod) *Status
 }
 
+// FilterPlugin is an interface for Filter plugins. These plugins are called at the
+// filter extension point for filtering out hosts that cannot run a pod.
+// This concept used to be called 'predicate' in the original scheduler.
+// These plugins should return only "Success" or "Unschedulable" in Status.code.
```

Comment:
"Error" is also a valid Status.code for these plugins.

Reply:
Done